modules/auth/auth/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">Auth</string>
+    <string name="app_name2">Auth</string>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kgmyshin/annict-android/blob/9f7d33eeb56630a8dca6454a13087b7b0625ab7c/modules/auth/auth/src/main/res/values/strings.xml#L3"></span>The resource <code>R.string.app_name2</code> appears to be unused</td>
    </tr>
  </tbody>
</table>